osl/interface_linux.go

```diff
@@ -62,6 +63,13 @@ func (i *nwIface) Master() string {
 	return i.master
 }
 
+func (i *nwIface) MacAddress() net.HardwareAddr {
```

Comment:
It is not just this function. but we need to stub a bunch of other existing interface methods that are not implemented.
We can handle it in another PR.